packages/vue-client/src/assets/main.css

```diff
@@ -113,4 +113,4 @@ a,
 .info-label {
   font-weight: bold;
   margin-right: 0.5em;
-}
\ No newline at end of file
+}
```

Comment:
Why delete the empty line at the end of that file? Was that some auto formatting?

Reply:
I don't know to be honest. Should I revert it?

Reply:
Maybe I misunderstood. I think GitHub is saying the `}` without `\n` was removed and a `}\n` was added, but the newline isn't visible in GitHub. Anyway, if the auto format thinks it should be this way, there's no need to revert it.